app/connectors/OnlineTestClient.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.CSRHttp
+import connectors.ExchangeObjects._
+import connectors.OnlineTestClient.{ OnlineTestNotFound, PdfReportNotFoundException }
+import models.ApplicationData.ApplicationStatus.ApplicationStatus
+import models.UniqueIdentifier
+import play.api.Play.current
+import play.api.libs.iteratee.Iteratee
+import play.api.libs.json.Json
+import uk.gov.hmrc.play.http._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+trait OnlineTestClient {
```

Comment:
Is OnlineTestClient the new  CubiksGatewayConnector? Or do we still keep CubiksGatewayConnector?